modules/system-view/README.md

```diff
@@ -0,0 +1,3 @@
+# System view module
+
+This module provides implementation of system views.
```

Comment:
Let's add an explanation how to create and register a system view.
Or create a ticket to fix README.

Reply:
Filed an issue https://issues.apache.org/jira/browse/IGNITE-20304.